akka-docs/src/main/paradox/stream/operators/Source/completionStage.md

```diff
@@ -18,8 +18,17 @@ Send the single value of the `CompletionStage` when it completes and there is de
 If the `CompletionStage` completes with `null` stage is completed without emitting a value.
 If the `CompletionStage` fails the stream is failed with that exception.
 
+@@@div { .group-scala }
+
 For the corresponding operator for the Scala standard library `Future` see @ref:[future](future.md).
 
+@@@
```

Comment:
Since it won't be possible to switch between languages in this page (done with snippet tabs and there only is a Java sample here) I think it's better to leave this as always shown.

Another aspect is that it's not impossible that you call a Java API from your Scala app and end up with a `CompletionStage` (the other way is perhaps less likely but also possible).

Reply:
There's a java/scala dropdown in the top-left of the page ;) - but might be clearer indeed.